app/views/artists/appointments/_show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h3><%= appointment_date %></h3>
+<table>
+  <% appointments.each do |appointment| %>
+    <tr>
+      <td><%= appointment.description %></td>
+      <td><%= link_to 'Edit', "/users/appointments/#{appointment.id}/edit", remote: true %></td>
+      <td><%= link_to 'Delete', "/users/appointments/#{appointment.id}", method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
`<td><%= link_to 'Delete', "/artists/appointments/#{appointment.id}", method: :delete, data: { confirm: 'Are you sure?' } %></td>`

Reply:
Ah thank you so much for your thorough review! It was so hard tracking down all instances of user and replacing with artist, and I think I forgot to check the updated master branch for all user instances. Woof!